config.yaml

```diff
@@ -35,6 +35,7 @@ teams:
   members:
   - jwickers
   - rybalko
+  - Vadneyk
```

Comment:
Hi @ryjones,
I think there's a small typo here, the final `K` should be uppercase (CLOWarden is complaining about this :slightly_smiling_face:).

Thanks!